setup.py

```diff
@@ -280,7 +280,7 @@ def get_extensions():
         if is_rocm_pytorch or torch.cuda.is_available() or os.getenv(
                 'FORCE_CUDA', '0') == '1':
             if is_rocm_pytorch:
-                define_macros += [('HIP_DIFF', None)]
+                define_macros += [('MMCV_WITH_HIP', None)]
```

Comment:
Why do we remove `HIP_DIFF`?

Reply:
https://github.com/open-mmlab/mmcv/blob/master/setup.py#L283

This might be a comparison error...